cloud/openstack/cmd/openstack-machine-controller/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reproducible builder image
+FROM golang:1.10.0 as builder
+WORKDIR /go/src/sigs.k8s.io/cluster-api
+# This expects that the context passed to the docker build command is
+# the cluster-api directory.
+# e.g. docker build -t <tag> -f <this_Dockerfile> <path_to_cluster-api>
+COPY . . 
+
+RUN CGO_ENABLED=0 GOOS=linux go install -a -ldflags '-extldflags "-static"' sigs.k8s.io/cluster-api/cloud/openstack/cmd/openstack-machine-controller
+
+# Final container
+FROM debian:stretch-slim
+RUN apt-get update && apt-get install -y ca-certificates openssh-server && rm -rf /var/lib/apt/lists/*
```

Comment:
why do we need openssh-server? (guess needed only during development). can we please remove this?

Reply:
looks like we may need the `ssh` executable, if we need just that we can use `openssh-client` instead of the server. right?